package.json

```diff
@@ -5,12 +5,16 @@
   "private": true,
   "dependencies": {
     "@craco/craco": "^3.4.0",
+    "@types/react-dnd": "^3.0.2",
     "antd": "^3.13.0",
+    "classcat": "^3.2.5",
```

Comment:
Do we really need this..? Seems like a bunch of bloat

Reply:
I really like using this for conditional classes, it's 307 bytes which is negligible & its readability is awesome IMO.

For 1 property it's arguably not that different but it's a nice way to avoid a lot of logic in the markup

```
<div className={`alwaysTrue ${condition ? 'sometimesTrue' : ''} ${otherCondition && rareCondition ? 'rarelyTrue' : ''}`}>
```

vs

```
const classNames = cc({
    alwaysTrue: true,
    sometimesTrue: condition,
    rarelyTrue: otherCondition && rareCondition
});

<div className={classNames}>
```